### PR TITLE
[dmn]Removes c4 from pubbystation and stetchkin from contrabrand locker spawners, removes illegal tech from syndicate playing cards

### PR DESCRIPTION
**Pull request by improvedname**
Sorry gamer rushing armory for free illegal tech(which has no downsides to deconstruct and isnt at all an accomplishment like locked nodes should be (see alien/bepis tech)

Also removes illegal tech from playing cards since its a common find in smuggler satchels/maintloot

Unlocking illegal tech should be done by dunking traitors not rushing easily accesible areas

Changelog
🆑 Improvedname
balance: Removes stetchkin from armory lootspawn
del: Removes c4 from pubbystation contrabrand locker
/🆑

### CODE REVIEW

**Comment by improvedname** [comment]
i cant believe i still failed to mapmerge 😳

**Comment by bobbahbrown** [comment]
make a stetchkin variant that doesn't add illegal tech and put that in the locker improve not remove

**Comment by bobbahbrown** [comment]
removing c4 is ok though

**Comment by trollbreeder** [comment]
Why does kilo have a seperate armory spawner?

**Comment by HomocidlePancakes** [comment]
There was a PR for a 10mm m1911. Got closed because it was bloat or whatever. This seems like a nice reason to readd it.

**Comment by HomocidlePancakes** [comment]
#43407

**Comment by Shadowflame909** [comment]
Why have a contraband locker if you don't want cheap contraband in there.

Hello?

**Comment by improvedname** [comment]
because the contrabrand thats inside them rn gives free illegal tech?? which defeats the whole point of illegal node being ***locked?***
Contrabrand is not only limited to syndicate technology
Its also Ied's stunprods illegal posters illegal ciggerates.

Not everything has to be about fucking illegal weaponry/armor
Like lmfao whats even up with you sec main powergays

**Comment by Shadowflame909** [comment]
Then subtype it, as was intended to happen in a previous PR specifically for the playing cards. But the current rework of embedding prevent that.


I just don't think a syndicate armory locker, with the goal of giving sec some tiny traitorous trinkets. Should lose those trinkets but keep the empty barren locker.

Sec has a whole room of evidence lockers, so it'd be best to just remove it entirely. Why keep useless things after making them useless..

**Comment by HomocidlePancakes** [comment]
https://github.com/tgstation/tgstation/pull/43407 Just replace stechkin with this.

**Comment by Iamgoofball** [comment]
``Also removes illegal tech from playing cards since its a common find in **smuggler satchels**``

..So, how smuggler satchels are supposed to be used?????????

**Comment by HomocidlePancakes** [comment]
You always get, like, 10 of smugglers satchels spawned every round.

**Comment by Iamgoofball** [comment]
Yes???? Because they stick around the map????

He's talking about ones that someone intentionally put cards into and hid somewhere.

**Comment by necromanceranne** [comment]
They're effectively extra storage space @Iamgoofball 

They spawn with contraband from the contraband crate ordered from cargo. Which includes a few 1tc items. They are also normal sized and carry a lot of items, which makes them excellent for packing gear into your bag.

**Comment by HomocidlePancakes** [comment]
@Iamgoofball In october/december 2018 I’ve abused the shit out of them. Every shift I would be running around with carp, nukie uplinks, wizard spell books (the 10 point ones), ekatanas and etc. Because of that, they were made no longer persist between rounds, but they do spawn with some trash items, like drugs or syndicards.

**Comment by Iamgoofball** [comment]
>Because of that, they were made no longer persist between rounds
the fuck is the point of them then?

**Comment by HomocidlePancakes** [comment]
They have drugs/stims, that’s pretty much it. I also use them for mousetrap bombs that fit in a backpack.

**Comment by Tlaltecuhtli** [comment]
they are pretty much ghetto boh with some free trinkets

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 1]
```diff
@@ -46,29 +46,18 @@
...
-				/obj/item/gun/ballistic/shotgun/automatic/combat = 5,
-				/obj/item/gun/ballistic/automatic/pistol/deagle,
-				/obj/item/gun/ballistic/revolver/mateba
+				/obj/item/grenade/iedcasing,
+				/obj/item/grenade/smokebomb,
+				/obj/item/poster/random_contraband,
+				/obj/item/storage/fancy/cigarettes/cigpack_syndicate,
+				/obj/item/storage/fancy/cigarettes/cigpack_mindbreaker,
+				/obj/item/melee/baton/cattleprod,
+				/obj/item/toy/cards/deck/syndicate,
```
Comment (Fikou) [inline code/game/objects/effects/spawners/lootdrop.dm]: this shit doesn't belong in the ARMORY

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 2]
```diff
@@ -46,29 +46,18 @@
 
 /obj/effect/spawner/lootdrop/armory_contraband
 	name = "armory contraband gun spawner"
```
Comment (Fikou) [inline code/game/objects/effects/spawners/lootdrop.dm]: its still called the gun spawner

**Review by Fikou — COMMENTED** [review]
a

File: code/game/objects/effects/spawners/lootdrop.dm (hunk 1 above)
Comment (improvedname) [reply code/game/objects/effects/spawners/lootdrop.dm]: it isnt supposed to be usefull you uncivilized powercreeper its a CONTRABRAND locker for contrabrand not for fucking guns you already have the whole fucking armoury

**Comment by Shadowflame909** [comment]
Improvedname, you can become a wizard by getting lucky enough to find a d20 in maint. Antags shouldn't be the only one with random rng changes of getting OP shit. This nerf PR is a nerf to just nerf.
Tldr: If on random occassions, your spooky scary OP antags can roll romerol for 15 tc instead of 25. Or get more then 20 tc worth of junk with those surplus crates. Sometimes an xenomorph will spawn in xenobio at roundstart, sec will spawn with a traitor gun, and a pulse rifle will spawn in the middle of space under a random tile that you have to crowbar.

This RNG is before you.

**Comment by Spepbuthewranglesgithub** [comment]
ah yes, another fun ban PR that will get merged anyways, resulting in rounds becoming _slightly_ more mundane.

_its all a ploy by the orange man to get more people to abandon byond so he has a less shitty platform to work with its all plan-_

**Comment by trollbreeder** [comment]
Merge conflicts with the meta medbay rework PR. Feex.

**Comment by ExcessiveUseOfCobblestone** [comment]
why does a gun need to spawn in there? That's where you're suppose to put the naughty stuff IN, not for it to spawn with naughty stuff you take OUT lmao.

**Comment by Shadowflame909** [comment]
@ExcessiveUseOfCobblestone 

We have a whole evidence locker room for that.

It's just another RNG spawner, like the ones antags get via discounts or surplus crates.

We have a shitload of these. This is just securities.

**Comment by ExcessiveUseOfCobblestone** [comment]
evidence locker isn't ID locked and may even allow lawyers to enter the area vs. armory where not even officers aren't normally allowed in. Evidence is where you throw assistant toolbelts after they broke in somewhere, Contraband Locker is for stuff that would nail you capital offenses (traitor gear, dismantled ttvs, etc.). It's why there's traitor guns in there to begin with lol.

**Comment by Shadowflame909** [comment]
Cobby you and I both know any antag weaponry sec gets their hands on stays in secs hands.

It's just the security departments meta breaker. Like how strange seeds are services. The roundstart xeno egg is sciences, and the deca crate is cargos.

**Comment by ExcessiveUseOfCobblestone** [comment]
I'm aware of what happens in practice, i'm just telling you it's purpose of existing in the first place.

**Comment by Shadowflame909** [comment]
It being called:

> /obj/effect/spawner/lootdrop/armory_contraband
> 	name = "armory contraband gun spawner"

Tells me the coder for this had different ideas then you do now.

**Comment by OdoSendai** [comment]
A removal/nerf PR justified by hand-wringing about game balance that never has any round impact in practice.  Speed merge please.

**Comment by ExcessiveUseOfCobblestone** [comment]
We're talking about the locker, not the spawner that handles spawning the guns...

Ofc the gun spawning effect would, in fact, spawn guns. I never said the gun spawner WASN'T suppose to spawn guns. It shouldn't exist in the first place.

**Comment by kriskog** [comment]
Ping me or ask a maintainer for a reopen when you can work on this again
